Neos.Media.Browser/Resources/Public/JavaScript/Main.js

```diff
@@ -236,27 +236,27 @@ if (undefined === 'production') {
 /* 4 */
 /***/ (function(module, exports) {
 
-var g;
-
-// This works in non-strict mode
-g = (function() {
-	return this;
-})();
-
-try {
-	// This works if eval is allowed (see CSP)
-	g = g || Function("return this")() || (1,eval)("this");
-} catch(e) {
-	// This works if the window reference is available
-	if(typeof window === "object")
-		g = window;
-}
-
-// g can still be undefined, but nothing to do about it...
-// We return undefined, instead of nothing here, so it's
-// easier to handle this case. if(!global) { ...}
-
-module.exports = g;
+var g;
+
+// This works in non-strict mode
+g = (function() {
+	return this;
+})();
+
+try {
+	// This works if eval is allowed (see CSP)
+	g = g || Function("return this")() || (1,eval)("this");
+} catch(e) {
+	// This works if the window reference is available
+	if(typeof window === "object")
+		g = window;
+}
+
+// g can still be undefined, but nothing to do about it...
+// We return undefined, instead of nothing here, so it's
+// easier to handle this case. if(!global) { ...}
+
+module.exports = g;
```

Comment:
Is this change related?

Reply:
Huh? No, definitely not. Where does this come from?

Reply:
Trailing CR are being removed:
<img width="912" alt="Screenshot 2020-08-11 at 18 57 14" src="https://user-images.githubusercontent.com/95873/89926004-88200080-dc04-11ea-85a0-40c979ad4c44.png">
Thats a good thing.